app/builders/v2/report_builder.rb

```diff
@@ -74,7 +74,7 @@ def get_grouped_values(object_scope)
       :created_at,
       default_value: 0,
       range: range,
-      permit: %w[day week month year],
+      permit: %w[day week month year hour],
```

Comment:
@sojan-official I am permitting `hour` here, I need it for the heatmap. Is it okay to do so?